examples/Test Sequencer/generic dut/sequencer_generic_dut.py

```diff
@@ -0,0 +1,147 @@
+            if not os.path.isabs(path):
+                config_root_dir = os.path.split(config_path)[0]
+                path = os.path.normpath(os.path.join(config_root_dir, path))
+            print("Loading Project...")
+            try:
+                project = app_reference.open_project(path)
+            except FlexLoggerError:
+                if not os.path.isfile(path):
+                    raise FileNotFoundError(
+                        "\n\nCheck CSV file. Cannot find FlexLogger project at path:\n{}".format(
+                            path
+                        )
+                    )
+                else:
+                    raise
+            print("Project Loaded\n")
+            test_session = project.test_session
+            print("Starting Test...")
+            start_test(test_session)
+            print("Test Started\n")
+            dut_control("changeState", state0)
+            display_elapsed_test_time(time0, test_session)
+            dut_control("changeState", state1)
+            display_elapsed_test_time(time1, test_session)
+            print("Pause Test")
+            test_session.pause()
+            dut_control("changeState", state2)
+            time.sleep(5)
+            print("Resume Test")
+            test_session.resume()
+            display_elapsed_test_time(time2, test_session)
+            dut_control("changeState", state3)
+            display_elapsed_test_time(time3, test_session)
+            test_session.stop()
+            print("Test Completed.\n\nClosing Project...")
+            project.close()
+            print("Project Closed\n")
+    return
+
+
+def start_test(
+    test_session, retries=3
+):  # This is a workaround for the FlexLogger API .start() method bug
+    for retry in range(retries):
+        test_session.start()
+        if test_session.state.value == 2:  # Running
+            return
+        elif retry < retries - 1:
+            print("Waiting for test session to start. Retry: {}".format(retry + 1))
+            time.sleep(1)
+            continue
+        else:
+            raise RuntimeError("display_time_elapsed: Test did not start within timeout")
+
+
+def dut_control(command, state="error"):
+    if command == "powerOn":
+        print("DUT Powered ON\n")
+    elif command == "powerOff":
+        print("DUT Powered OFF\n")
+    elif command == "changeState":
+        print("Change DUT to State " + str(state))
+    elif command == "updateFirmware":
+        print("Simulate Downloading DUT Firmware...")
+        time.sleep(2)
+        print("DUT Firmware Downloaded Successfully\n")
+    else:
+        print("Error: Incorrect DUT Command\n")
+    return
+
+
+def display_elapsed_test_time(total_time, test_session):
+    timestamp = test_session.elapsed_test_time
+    base_time = (
+        (timestamp.days * 24 * 60 * 60) + timestamp.seconds + (timestamp.microseconds / 1000000)
+    )  # Report time in seconds
+    time_diff = 0
+    total_time_int = int(total_time)
```

Comment:
Given that we're only sleeping for .1 second at a time, I think we should use `float()` instead of `int()` here. Although I guess if we know people are only going to be using integers in the .csv file, this is OK.

Reply:
Agreed. Updated to float.